src/classes/BGE_ConfigurationWizard_CTRL_TEST.cls

```diff
@@ -39,152 +39,150 @@
-                '"contactMatchingRule":"Firstname,Lastname,Email",' +
-                '"donationMatchingBehavior":"'+ BDI_DataImport_API.RequireBestMatch + '",' +
-                '"donationMatchingRule":["' + UTIL_Namespace.StrAllNSPrefix('Donation_Amount__c').toLowerCase() + '"],' +
-                '"donationDateRange":1,' +
-                '"processUsingScheduledJob":false,' +
-                '"templateFields":null,' +
-                '"labels":null' +
-                '}';
+    private static void testSaveNewBatch() {
+
+        String batchInfoJSON = '{' +
+            '"name":"Test 4",' +
+            '"id":null,' +
+            '"description":"Test Record",' +
+            '"requireTotalMatch":false,' +
+            '"batchProcessSize":5,' +
+            '"runOpportunityRollupsWhileProcessing":false,' +
+            '"contactMatchingRule":"Firstname,Lastname,Email",' +
+            '"donationMatchingBehavior":"'+ BDI_DataImport_API.RequireBestMatch + '",' +
+            '"donationMatchingRule":["' + UTIL_Namespace.StrAllNSPrefix('Donation_Amount__c').toLowerCase() + '"],' +
+            '"donationDateRange":1,' +
+            '"processUsingScheduledJob":false,' +
+            '"availableFields":null,' +
+            '"labels":null' +
+        '}';
         String activeFieldsJSON = BGE_ConfigurationWizard_CTRL.getDefaultActiveFields();
         Test.startTest();
-        String jsonModel = BGE_ConfigurationWizard_CTRL.saveRecord(sObjectName, templateInfoJSON, activeFieldsJSON);
-        BGE_ConfigurationWizard_CTRL.BGEDetailsModel templateDetails = (BGE_ConfigurationWizard_CTRL.BGEDetailsModel) JSON.deserialize(jsonModel, BGE_ConfigurationWizard_CTRL.BGEDetailsModel.class);
+        String jsonModel = BGE_ConfigurationWizard_CTRL.saveRecord(batchInfoJSON, activeFieldsJSON);
+        BGE_ConfigurationWizard_CTRL.BGEDetailsModel batchDetails = (BGE_ConfigurationWizard_CTRL.BGEDetailsModel) JSON.deserialize(jsonModel, BGE_ConfigurationWizard_CTRL.BGEDetailsModel.class);
         Test.stopTest();
 
         System.assertNotEquals(null, jsonModel, 'JSON string of entire model should be returned.');
-        System.assertEquals('Test 4', templateDetails.name, 'Template details name should be Test 4');
-        System.assertEquals('Test Record', templateDetails.description, 'Template details description should be Test Record');
-        System.assertEquals(false, templateDetails.requireTotalMatch, 'Require total match should be false');
-        System.assertEquals(BGE_ConfigurationWizard_CTRL.getDefaultActiveFields(), templateDetails.activeFields, 'Active Fields should match default');
-        System.assertNotEquals(null, templateDetails.id, 'Template details should not be null');
+        System.assertEquals('Test 4', batchDetails.name, 'Template details name should be Test 4');
+        System.assertEquals('Test Record', batchDetails.description, 'Template details description should be Test Record');
```

Comment:
Can change here to use just 'Batch name' and 'Batch description' instead of 'Template details'